src/brain/issueLabelHandler/route.ts

```diff
@@ -67,6 +67,36 @@ function shouldLabelBeRemoved(labelName, target_name) {
   );
 }
 
+async function getReadableTimeStamp(timeToTriageBy, productAreaLabelName) {
```

Comment:
thought this would be a better place to put this, since it's above all the functions that listen to webhooks